test/modeling/test_processor.py

```diff
@@ -296,3 +296,28 @@ def test_dataset_from_dicts_qa_labelconversion(samples_path, caplog=None):
                         12,
                         12,
                     ], f"Processing labels for {model} has changed."
+
+
+def test_dataset_from_dicts_auto_determine_max_answers(samples_path, caplog=None):
```

Comment:
Let's add the integration test marker here (we need to add `import pytest` for this):
```suggestion

@pytest.mark.integration
def test_dataset_from_dicts_auto_determine_max_answers(samples_path, caplog=None):
```